src/templates/labels_form_inner_template.html

```diff
@@ -23,14 +23,20 @@
                         <a href="#" data-label="[key-values]">Key-Value Pairs</a>
                     </li>
                     <li title="Add a label that shows the current T-index">
-                        <a href="#" data-label="[time-index]">Time (index)</a>
+                        <a href="#" data-label="[time-index]">Time (T-index)</a>
+                    </li>
+                    <li class="add_time_label" title="Label shows time in milliseconds">
+                        <a href="#" data-label="[time-millisecs]">Time (millisecs)</a>
```

Comment:
I might I have raised the issue on a previously PR before but I would strongly recommend against the usage of custom abbreviation. Any reason not to use either the SI term `milliseconds` or the SI abbreviation `ms`?

Reply:
agree 
The recommendations should be used